docs/modules/deployment/pages/system-scalability/time-series-storage.adoc

```diff
@@ -4,7 +4,7 @@
 For different scenarios it is useful to switch to a different time series storage.
 The following implementations are installed by default:
 
-.Supported Time Series Databasees
+.Supported Time Series Databases
```

Comment:
Fixed in a later branch

Reply:
I kept this change in (for now), as it was still present in `foundation-2021`. Should I remove it as well?

Reply:
This file will be updated via https://github.com/OpenNMS/opennms/pull/3702

Reply:
I've removed this change from the PR.